subsystem/wrist.py

```diff
@@ -99,10 +101,11 @@ def get_wrist_angle(self):
         #     (self.wrist_abs_encoder.getPosition())
         # )
 
-    def note_detected(self) -> bool:
-        # return self.distance_sensor.getVoltage() < config.feeder_sensor_threshold
-        # return False
-        return self.distance_sensor.getVoltage() > config.feeder_sensor_threshold
+    def detect_note_first(self) -> bool:
+        return not self.beam_break_first.get()
+    
+    def detect_note_second(self) -> bool:
+        return not self.beam_break_second.get()
```

Comment:
You probably still want to have this method of note_detected, since that could be a combo of both of the beam breaks, or maybe returning one or the other. We'll see, but its easier to update everything else that uses it by just having that method